data/transactions/verify/txn.go

```diff
@@ -92,6 +88,20 @@ func stxnAsyncVerify(arg interface{}) interface{} {
 }
 
 func stxnVerifyCore(s *transactions.SignedTxn, proto *config.ConsensusParams) error {
+	// Check for non-empty fields that are not yet enabled
```

Comment:
This is a nit, but shouldn't these checks go into `Transaction.WellFormed`?

Reply:
Actually, it turns out a lot of these checks were done in `WellFormed` (I missed that `WellFormed` was getting called in `Txn`). The only missing check there was the `Group` check. So I added that and removed the extra ones I added.